application/eu-central-1/terraform.tfvars

```
@@ -176,7 +186,15 @@ k8s_aws_ssh_keypair_name = ""
 
 operations_aws_account_number = "123456789012"
 
-# Cross-account role to assume before deploying the cluster. Example:
+# Cross-account role to assume before deploying the cluster.
+# For  auto_IAM_mode = false
+# Example:
 # iam_cross_account_role_arn = "arn:aws:iam::210987654321:role/KopsCrossAccount"
+# For auto_IAM_mode = true
+# Schema:  arn:aws:iam::{application_aws_account_number}:role/KENTRIKOS_{region}_{product_domain_name}_{environment_type}_CrossAccount
+# Example: arn:aws:iam::210987654321:role/KENTRIKOS_eu-central-1_demo_test_CrossAccount"
+
+iam_cross_account_role_arn = "arn:aws:iam::c:role/KopsCrossAccount"
```

Please update arn with full role format
```suggestion
iam_cross_account_role_arn = "arn:aws:iam::210987654321:role/KopsCrossAccount"
```